src/common/Layout/Layout.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect } from 'react';
+import { useLocation, useNavigate } from 'react-router';
+import Header from '../../components/Header/Header';
+import { isLogedIn } from '../../helpers/localStorage';
+
+function Layout({ children }) {
+	const location = useLocation();
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		console.log(location);
+		if (
+			(location.pathname !== '/login' ||
+				location.pathname !== '/registration') &&
+			!isLogedIn()
+		) {
+			navigate('/login');
+		}
+	}, []);
+
+	return (
+		<>
+			<header>
+				<Header />
+			</header>
```

Comment:
Sounds a good idea to include semantic tag <header> on your <Header /> component, right?

Reply:
yeah you are right